Tests/bootstrap.php

```diff
@@ -8,7 +8,8 @@
 if (file_exists(__DIR__ . '/../vendor/autoload.php')) {
     $loader = require __DIR__ . '/../vendor/autoload.php';
 
-    AnnotationRegistry::registerLoader('class_exists');
+    if(method_exists(AnnotationRegistry::class, 'registerLoader'))
+    {AnnotationRegistry::registerLoader('class_exists');}
```

Comment:
Just a small formatting change
```suggestion
    if (method_exists(AnnotationRegistry::class, 'registerLoader')) {
        AnnotationRegistry::registerLoader('class_exists');
    }
```